vowpalwabbit/parse_regressor.cc

```diff
@@ -349,8 +350,18 @@ void save_load_header(
         else  // < VERSION_FILE_WITH_INTERACTIONS
         {
           // pairs and triples may be restored but not reflected in interactions
-          all.interactions.insert(std::end(all.interactions), std::begin(all.pairs), std::end(all.pairs));
-          all.interactions.insert(std::end(all.interactions), std::begin(all.triples), std::end(all.triples));
+          std::vector<std::vector<uint8_t>> newpairs_(all.pairs.size());
```

Comment:
This doesn't fit in with naming conventions

```suggestion
          std::vector<std::vector<uint8_t>> new_pairs(all.pairs.size());
```

Reply:
Updated